src/__tests__/Col.spec.js

```diff
@@ -35,21 +35,18 @@ describe('Col', () => {
     expect(wrapper.hasClass('col-wtf-1')).toBe(true);
     expect(wrapper.hasClass('order-wtf-2')).toBe(true);
     expect(wrapper.hasClass('offset-wtf-4')).toBe(true);
-    expect(wrapper.hasClass('col')).toBe(true);
```

Comment:
I would explicitly check for it to not have the class (change `true` to `false`) as with the test as it is now, `col` could be added back and nothing would break.

Reply:
reverted and changed to `true`